go/vt/topo/consultopo/server.go

```diff
@@ -37,13 +40,32 @@ import (
 )
 
 var (
-	consulAuthClientStaticFile = flag.String("consul_auth_static_file", "", "JSON File to read the topos/tokens from.")
-	// serfHealth is the default check from consul
-	consulLockSessionChecks = flag.String("topo_consul_lock_session_checks", "serfHealth", "List of checks for consul session.")
-	consulLockSessionTTL    = flag.String("topo_consul_lock_session_ttl", "", "TTL for consul session.")
-	consulLockDelay         = flag.Duration("topo_consul_lock_delay", 15*time.Second, "LockDelay for consul session.")
+	// flag vars
+	consulAuthClientStaticFile = ""
```

Comment:
for things that default to the zero value for their type (`string` => `""`, `int` => `0`, and so on), you can just do `var myVar string` (or `var myVar int`, etc)